SConstruct

```diff
@@ -1,4 +1,74 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
-# Python-only package
-scripts.BasicSConstruct("ci_ap", disableCc=True, noCfgFile=True)
+import os
+from SCons.Script import Environment, GetOption, Default
+from lsst.sconsUtils.utils import libraryLoaderEnvironment
+
+env = Environment(ENV=os.environ)
```

Comment:
It's usually dangerous to copy the entire environment into SCons, but I think I understand why you're doing it.  A comment to that effect would be good.

Reply:
Again, copied from ci_hsc, so I don't really know why it needs the whole thing.